crates/release_plz/src/args/release.rs

```diff
@@ -2,12 +2,13 @@ use std::path::PathBuf;
 
 use anyhow::Context;
 use clap::builder::{NonEmptyStringValueParser, PathBufValueParser};
-use release_plz_core::ReleaseRequest;
+use git_cmd::Repo;
+use release_plz_core::{GitBackend, GitHub, Gitea, ReleaseRequest, RepoUrl};
 use secrecy::SecretString;
 
-use super::local_manifest;
+use super::{local_manifest, release_pr::GitBackendKind};
 
-#[derive(clap::Parser, Debug)]
+#[derive(clap::Parser, Debug, Clone)]
```

Comment:
We don't want to clone the whole struct
```suggestion
#[derive(clap::Parser, Debug)]
```